reccmp/tools/project.py

```diff
@@ -123,7 +123,8 @@ def main():
         except RecCmpProjectException as e:
             logger.error("Project detection failed: %s", e.args[0])
 
-    parser.error("Missing command: create/detect")
+    else:
+        parser.error("Missing command: create/detect")
 
     # try:
```

Comment:
Is this commented code still relevant? Maybe it can be removed during this refactor?

Reply:
Yeah I think the argparse subparser will error earlier than this.

Reply:
We do get here if you run `reccmp-project` with no subcommand